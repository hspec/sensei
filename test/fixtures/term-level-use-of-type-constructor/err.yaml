version: '1.1'
ghcVersion: ghc-9.10.0
span:
  file: test/fixtures/term-level-use-of-type-constructor/Foo.hs
  start:
    line: 4
    column: 7
  end:
    line: 4
    column: 16
severity: Error
code: 1928
message:
- Illegal term-level use of the type constructor `CodeBlock'
- '  imported from `Text.Markdown.Unlit'' at test/fixtures/term-level-use-of-type-constructor/Foo.hs:2:29-37'
- |-
  Perhaps use `CodeBloc' (line 3)
  Add `CodeBlock' to the import list in the import of
  `Text.Markdown.Unlit'
  (at test/fixtures/term-level-use-of-type-constructor/Foo.hs:2:1-38).
- |-
  In the expression: CodeBlock
  In an equation for `foo': foo = CodeBlock
hints: []
