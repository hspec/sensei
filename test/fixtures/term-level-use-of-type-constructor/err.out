test/fixtures/term-level-use-of-type-constructor/Foo.hs:4:7: error: [GHC-01928]
    • Illegal term-level use of the type constructor `CodeBlock'
    • imported from `Text.Markdown.Unlit' at test/fixtures/term-level-use-of-type-constructor/Foo.hs:2:29-37
    • Perhaps use `CodeBloc' (line 3)
      Add `CodeBlock' to the import list in the import of
      `Text.Markdown.Unlit'
      (at test/fixtures/term-level-use-of-type-constructor/Foo.hs:2:1-38).
    • In the expression: CodeBlock
      In an equation for `foo': foo = CodeBlock

